plasmapy/simulation/abstractions.py

```diff
@@ -54,3 +54,142 @@ class AbstractTimeDependentSimulation(AbstractSimulation):
     """
 
     pass
+
+
+class AbstractNormalizations(ABC):
+    """
+    An abstract base class to represent the normalizations of systems of
+    equations describing plasmas.
+
+    Warnings
+    --------
+    This interface is unstable and is subject to change.
+    """
+
+    @property
+    @abstractmethod
+    def current_density(self) -> u.A / u.m ** 2:
```

Comment:
Are these methods suppose to be the normalized value, un-normalized value, or the the normalizing constant?

Reply:
These are to be the normalizing constant, which I've most often heard referred to as the normalization.  Maybe I should clarify that in the class docstring...